package com.example.demo.board.notice;

import java.util.Optional;

import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;
import org.springframework.data.jpa.repository.JpaRepository;

public interface NoticeRepository extends JpaRepository<NoticeVO, Long> {
	
//	public NoticeFileVO save(NoticeFileVO noticeFileVO) throws Exception;
	
}
